dts/bindings/timer/nordic,nrf-grtc.yaml

```diff
@@ -4,18 +4,53 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-description: Nordic GRTC (Global RTC)
+description: |
+  Nordic GRTC (Global RTC)
+
+      Example of using clock outputs:
+        &grtc {
+          pinctrl-0 = <&grtc_default>;
+          pinctrl-1 = <&grtc_sleep>;
+          pinctrl-names = "default", "sleep";
+          clkout-fast-frequency = <8000000>;
+          clkout-32k-frequency = <32768>;
+          /* In case of nRF54H20 devices: */
+          nordic,clockpin-enable = <NRF_FUN_GRTC_CLKOUT_FAST>;
+        };
 
 compatible: "nordic,nrf-grtc"
 
 include:
   - "base.yaml"
   - "nordic,split-channels.yaml"
+  - "pinctrl-device.yaml"
+  - "nordic-clockpin.yaml"
 
 properties:
   reg:
     required: true
 
+  pinctrl-0:
+    required: false
+
+  pinctrl-names:
+    required: false
+
+  base-frequency:
+    type: int
+    description: base clock frequency.
+    required: true
+
+  clkout-fast-frequency:
+    type: int
+    description: Fast output clock frequency.
+
+  clkout-32k-frequency:
+    type: int
+    description: Slow output clock frequency.
+    enum:
+      - 32768
```

Comment:
`const: 32768`

btw, does this prop make sense? its name contains the actual frequency...

Reply:
Changed to boolean